src/olympia/landfill/serializers.py

```diff
@@ -103,25 +130,36 @@ def create_named_addon_with_author(self, name, author=None):
         if author is None:
             author = user_factory()
         try:
-            generate_user(author)
+            user = UserProfile.objects.create(username=author, email=f'{author}@email.com')
+            user.id = 4757633
```

Comment:
You will have to use `user.update(id=settings.4757633)` ideally to not hardcode the id.